middle_end/flambda2/types/reify.ml

```diff
@@ -384,7 +384,10 @@ let reify ~allowed_if_free_vars_defined_in ~var_is_defined_at_toplevel
       | _ :: _ -> (
         match element_kind with
         | Unknown -> try_canonical_simple ()
-        | Known element_kind -> (
+        | Bottom ->
+          (* Note: we could use [Lift Empty_array] here *)
```

Comment:
Same again, make this a `CR-someday` as a potential future improvement.

Reply:
Done